iothub/device/src/ClientFactory.cs

```diff
@@ -23,7 +23,7 @@ internal class ClientFactory
         /// <summary>
         /// Create an Amqp InternalClient from individual parameters
         /// </summary>
-        /// <param name="hostname">The fully-qualified DNS hostname of IoT Hub</param>
+        /// <param name="hostname">The fully-qualified DNS host name of IoT Hub</param>
```

Comment:
The other overloaded methods also have hostname.

Reply:
just a test to run the pipeline. will close afterwards.